app/controllers/draws_controller.rb

```diff
@@ -31,6 +33,11 @@ def destroy
     handle_action(**result)
   end
 
+  def start
```

Comment:
this is super nit-picky but I feel like `start` is too ambiguous between opening up the draw to students and running the lottery -- `open` might be better for this transition?